CPT331.Data.Migration/Migrations/GeographyNTBalance.sql

```diff
@@ -0,0 +1,1096 @@
+128.999994 -21.683757000000007,
+128.9999886 -21.570819899999993,
+128.99998319999997 -21.457882800000007,
+128.99997779999995 -21.344945599999996,
+128.99997490000004 -21.06204500000001,
+128.99997739999998 -20.892081599999994,
+128.99998 -20.72211810000001,
+128.9999825 -20.552154599999998,
+128.999985 -20.3821912,
+128.99998749999997 -20.212227700000003,
+128.99999000000003 -20.042264200000005,
+128.99999179999998 -19.89591890000002,
+128.9999926 -19.872300800000005,
+128.9999951 -19.702337300000007,
+128.99999759999997 -19.532373799999995,
+129.00000010000002 -19.36241040000001,
+129.00000259999993 -19.19244690000001,
+129.0000052 -19.0224834,
+129.0000077 -18.852520000000002,
+129.0000112 -18.615752900000004,
+129.00001219999996 -18.5489494,
+129.0000131 -18.482145800000012,
+129.00001410000002 -18.41534230000001,
+129.00001509999998 -18.348538699999995,
+129.00001609999998 -18.281735099999995,
+129.00001699999999 -18.214931600000018,
+129.00001820000003 -18.13475130000002,
+129.00001840000002 -18.121374599999992,
+129.00001860000003 -18.107997900000004,
+129.00001880000002 -18.094621200000006,
+129.000019 -18.081244399999996,
+129.0000192 -18.067867700000008,
+129.0000194 -18.054490999999995,
+129.0000196 -18.041114299999993,
+129.000019 -17.969496200000002,
+129.00001820000003 -17.91125480000001,
+129.00001739999996 -17.85301340000002,
+129.0000166 -17.794772000000002,
+129.00001579999997 -17.736530700000007,
+129.000015 -17.678289300000003,
+129.0000142 -17.6200479,
+129.0000134 -17.561806500000007,
+129.0000126 -17.503565100000003,
+129.0000118 -17.4453237,
+129.00001099999997 -17.387082300000007,
+129.00001019999996 -17.328840900000017,
+129.0000094 -17.2705995,
+129.00000860000003 -17.21235820000002,
+129.0000078 -17.1541168,
+129.000007 -17.095875400000008,
+129.00000619999997 -17.037634000000004,
+129.000004 -16.8919099,
+129.00000259999993 -16.804427300000015,
+129.00000119999999 -16.71694460000002,
+128.99999970000002 -16.629462000000007,
+128.99999830000002 -16.541979299999998,
+128.99999689999999 -16.454496600000013,
+128.9999955 -16.367013999999998,
+128.9999941 -16.279531300000002,
+128.9999927 -16.192048700000015,
+128.9999912 -16.104566000000016,
+128.99998980000004 -16.01708340000002,
+128.9999872 -15.981731900000005,
+128.99998940000003 -15.981462400000007,
+128.99998839999998 -15.929600700000007,
+128.999987 -15.842117999999997,
+128.99998560000003 -15.754635399999998,
+128.9999842 -15.6671527,
+128.99998269999998 -15.579670100000014,
+128.9999823 -15.552343799999997,
+128.9999813 -15.492187400000017,
+128.99998040000003 -15.434136299999993,
+128.99997989999997 -15.404704800000005,
+128.99997849999997 -15.317222099999993,
+128.9999771 -15.229739399999996,
+128.99997560000003 -15.142256799999997,
+128.9999742 -15.054774100000012,
+128.9999728 -14.967291500000012,
+128.99997140000002 -14.879844200000006)))';
+DECLARE @NT GEOGRAPHY = @geom.MakeValid().STUnion(@geom.STStartPoint()).STAsText();
```

Comment:
NT turned into negative space when I applied the Geography type to these coords.

This line corrects the inversion of the shape.